dockstore-webservice/src/main/java/io/dockstore/webservice/core/SourceFile.java

```diff
@@ -371,4 +385,9 @@ public static class VerificationInformation {
 
         // There is no dbupdatedate because it doesn't work with @Embeddable nor @ElementCollection
     }
+
+    public enum FormEnum {
```

Comment:
Should probably code comment the enum and the enum values. 
Don't feel strongly about this, but `Form` doesn't really convey much to me aside from Google Forms, web forms, etc.
Perhaps "status" or "state"?

Reply:
Yeah, this one is tough.  "Status" and "state" are both vague, and suggest the possibility future change.  "Mode" is a previously-loaded term when it comes to files.  Maybe "representation", which could have values `COMPLETE` and `NONE`?

Reply:
I like representation better than form at least

Reply:
Not wild about `Form` either, although I can see why you chose it. I don't think I like `Representation` either -- I may be biased against it from its meaning in REST.

`SyncState` or `SyncStatus`? But that's not really applicable to hosted entries. `ContentState`?